src/controller/CHIPDeviceController.cpp

```diff
@@ -503,6 +503,16 @@ CHIP_ERROR DeviceController::ServiceEventSignal()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DeviceController::GetFabricId(uint64_t & fabricId)
```

Comment:
This file is not changed but I have worked on csg/test_event_3.1_fabric_id as base so all these changes are showing up in this PR.